README.md

```diff
@@ -17,7 +17,7 @@ Several applications are provides by default:
 
 ### Dual-IIR
 
-![Flow diagram](stabilizer_pid.svg)
+![Flow diagram](docs/assets/stabilizer_pid.svg)
```

Comment:
This diagram is outdated and inaccurate. Docs in this README should now point to the stabilizer user manual site.

Reply:
@SingularitySurfer could you spin a new diagram (sample rate 781 kHz) ? The existing one's source is the SVG in the repo if you need it. I used [draw.io](https://draw.io) back then.

Regarding including/showing images in cargo docs, it works just fine with relative links for all "local" crates (which is what we care about here) and with absolute links for everything else.

https://docs.rs/ndarray/0.11.2/ndarray/type.ArrayView.html#method.split_at
or
https://users.rust-lang.org/t/include-images-in-rustdoc-output/3487/4